<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="MainStackWidget_widget">
      <widget class="QWidget" name="VideoMatchPage_widget">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="VideoShow_vlayout">
          <item>
           <widget class="QWidget" name="VideoShow_widget" native="true">
            <property name="minimumSize">
             <size>
              <width>800</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Label_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="VideoOperate_vlayout" stretch="0,0,0">
          <item>
           <widget class="QListView" name="UsersList_listview">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="StartMatch_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>开始匹配</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="CloseVideo_btn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>结束视频</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SocialHall_widget">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="RoomsList_tableview"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="RoomOperate_vlayout">
          <item>
           <widget class="QPushButton" name="CreateRoom_btn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>创建房间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FlushList_btn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>刷新列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="JoinRoom_btn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>加入房间</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PageSelect_hlayout">
      <item>
       <widget class="QPushButton" name="VideoMatch_btn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>视频匹配</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SocialHall_btn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>社交大厅</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
